src/Components/AdvancedSearch/AdvancedSearch.stories.tsx

```diff
@@ -1,24 +1,44 @@
 import React from 'react';
 import { ComponentStory } from '@storybook/react';
-import { getDefaultStoryDecorator } from '../../Models/Services/StoryUtilities';
-import AdvancedSearchModal from './AdvancedSearchModal';
+import {
+    getDefaultStoryDecorator,
+    IStoryContext
+} from '../../Models/Services/StoryUtilities';
+import AdvancedSearch from './AdvancedSearch';
 import { IAdvancedSearchProps } from './AdvancedSearch.types';
+import MockAdapter from '../../Adapters/MockAdapter';
 
 const wrapperStyle = { width: '100%', height: '100vh', padding: 8 };
 
 export default {
     title: 'Components/AdvancedSearch',
-    component: AdvancedSearchModal,
+    component: AdvancedSearch,
     decorators: [getDefaultStoryDecorator<IAdvancedSearchProps>(wrapperStyle)]
 };
 
-type AdvancedSearchStory = ComponentStory<typeof AdvancedSearchModal>;
+type AdvancedSearchStory = ComponentStory<any>;
 
-const Template: AdvancedSearchStory = (args) => {
-    return <AdvancedSearchModal {...args} />;
+const Template: AdvancedSearchStory = (
+    args: IAdvancedSearchProps,
+    context: IStoryContext<IAdvancedSearchProps>
+) => {
+    return <AdvancedSearch {...args} theme={args.theme || context.theme} />;
```

Comment:
lets use the default decorator which already wraps the component with the context to be consistent in our stories so you can remove theme prop here: 
<img width="452" alt="image" src="https://user-images.githubusercontent.com/45217314/188013816-692af437-bbbf-440c-b182-aa4ba1d23da0.png">